include/buildpack.bash

```diff
@@ -94,6 +94,7 @@ buildpack-execute() {
 		exit 1
 	fi
 
+	cd "$build_path"
```

Comment:
Maybe we should execute this in another writeable directory. `heroku run ls -la` doesn't list the bin folder created by the grails bp.